assets/root-files/travis.yml

```diff
@@ -19,17 +21,16 @@ notifications:
   email: false
  
 install:
-  - if [[ $TRAVIS_PHP_VERSION != '5.6' && $TRAVIS_PHP_VERSION != 'hhvm' && $TRAVIS_PHP_VERSION != '7' ]]; then phpenv config-rm xdebug.ini;  fi
+  - if [[ $EXECUTE_TEST_COVERALLS != 'true' ]]; then phpenv config-rm xdebug.ini ; fi
```

Comment:
It's important append `|| return 0` for silent errors from PHP 7 and HHVM (they don't have xdebug.ini and I didn't found a command for ask about the existence)

`return` is used instead `exit` because the last one cut Travis-CI and the first one is valid because is executed in a script context execution